api_review/personpicture/PersonPictureAPI.md

```diff
@@ -0,0 +1,81 @@
+# Background
+
+Xaml controls are typically implemented using an element tree that’s defined in a template. 
+For example the template for a ComboBox control has a TextBlock in it to display the selected value.
+
+Several controls have properties that are only expected to be useful to these templates, 
+and so there’s a common pattern for the control to expose them in a TemplateSettings property. 
+For example [ComboBox.TemplateSettings](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.ComboBox.TemplateSettings) 
+is used by the ComboBox’s template. See [this doc page](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes) for more details. 
+
+This API update introduces this pattern to the PersonPicture control: 
+a new TemplateSettings property and associated PersonPictureTemplateSettings class.
+
+# Description
+
+Provides calculated values that can be referenced as TemplatedParent sources when defining templates 
+for a PersonPicture control. Not intended for general use.
+
+# API Details
+
+```
+namespace Microsoft.UI.Xaml.Controls
+{
+    [webhosthidden]
+    unsealed runtimeclass PersonPicture : Windows.UI.Xaml.Controls.Control
+    {
+        ...
+
+        PersonPictureTemplateSettings TemplateSettings { get; };

```

Comment:
Presumably this property needs a new contract?

Reply:
It would if it was shipping in the OS, but this is shipping out-of-band so we don't need to be ABI compatible version-to-version.